backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -14,9 +14,16 @@ You back up applications by creating a `Backup` custom resource (CR). See xref:.
 
 For more information about CSI volume snapshots, see xref:../../../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc#persistent-storage-csi-snapshots[CSI volume snapshots].
 
-:FeatureName: The `CloudStorage` API for S3 storage
+:FeatureName: The `CloudStorage` API, which automates the creation of a bucket for object storage,
 include::snippets/technology-preview.adoc[]
 
+[NOTE]
+====
+The `CloudStorage` API supports manually creating a `BackupStorageLocation` object specifying an existing S3 bucket. The `CloudStorage` API that creates an S3 bucket automatically, currently is only enabled for AWS S3 storage.
```

Comment:
Both of these sentences are a little confusing. They need a little rework. Here's my attempt:

```suggestion
The `CloudStorage` API supports manually creating a `BackupStorageLocation` object by specifying an existing S3 bucket. The `CloudStorage` API that creates an S3 bucket automatically is currently only enabled for AWS S3 storage.
```

Reply:
@stevsmit 
Thank you very much for your rewrite.
I took your suggestion as is for the first sentence and followed some of your directions with the second sentence. I also switched the order around so that it will be easier for the reader to understand what the current limitations are.
Thank you for your feedback.
Carmi